### PR TITLE
[Demo-express] Improve notification test

### PR DESCRIPTION
**Pull request by xinliux**
- Add Notification permission verification

Impacted tests(approved):new 0,update 1,delete 0
Unit test platform:[Android]
Unit test result summary:pass 1,fail 0,block 0

### CODE REVIEW

File: samples/Notifications/js/main.js [hunk 1]
```diff
@@ -39,7 +39,8 @@ jQuery(document).ready(function() {
 
 
 function showNotification() {
-  notification = typeof Notification != "undefined" ? new Notification("New Email Received", {
+  if(Notification.permission === "granted") {
```
Comment (zqzhang) [inline samples/Notifications/js/main.js]: Put a space before `(`

File: samples/Notifications/js/main.js [hunk 2]
```diff
@@ -47,7 +48,23 @@ function showNotification() {
       body: "Room 101",
       tag: "tom"
     });
-  notification.show();
+  } else if(Notification.permission !== 'denied') {
```
Comment (zqzhang) [inline samples/Notifications/js/main.js]: ditto

**Comment by xinliux** [comment]
@zqzhang  Please help merge this PR, thanks!

**Comment by zqzhang** [comment]
@jennycao please help merge.

**Pull request merged**